docs/integrations/keras.rst

```diff
@@ -1,112 +1,319 @@
-   :alt: Keras neptune.ai integration
+Introduction
+------------
 
+|Keras| is a high level interactive API that can be used to prototype, build and train neural networks. \
+Keras acts as an interface for the TensorFlow library, and in the latest version of Tensorflow, Keras was officially integrated under the name |tf.keras|.
+
+By the end of this example you will be able to: \
+
+* How to integrate Neptune with Keras to effectively track metrics and model training. 
+* How to easily reproduce experiments and aid collaboration in your project. 
+
+.. Note::
+
+   This integration works with library==tf.keras v0.24.1 and neptune-contrib v0.24.1 
+
+Links:
+|source code| |open in colab|
+
+.. |Keras| raw:: html
+
+    <a href="https://keras.io" target="_blank">Keras</a> 
+
+.. |tf.keras| raw:: html
+
+    <a href="https://www.tensorflow.org/api_docs/python/tf/keras" target="_blank">tf.keras</a>
+
+.. |link| raw:: html
+
+    <a href="https://www.pyimagesearch.com/2019/10/21/keras-vs-tf-keras-whats-the-difference-in-tensorflow-2-0" target="_blank">link</a>
+
+.. |open in colab| raw:: html
+
+    <a href="https://colab.research.google.com/drive/1ITLOePfJh9rNR1jis5gBES2InDo_8Iz4?usp=sharing" target="_blank">open in colab</a>
+
+
+.. |steps here| raw:: html
+
+    <a href="https://docs.neptune.ai/workspace-project-and-user-management/projects/create-project.html" target="_blank">open in colab</a>
+
+
+.. |signup| raw:: html
+
+    <a href="https://neptune.ai/register" target="_blank">open in colab</a>
+
+Quickstart
+==========
+
+**Step 1**
+
+Get a neptune account and your API token. You can |signup| here.
+
+**Step 2**
+
+Create a new project called spam-classifier. You can follow the |steps here| to create a new project. 
+
+**Step 3**
+
+In your environment (colab/jupyter/python script) and install neptune client library.
+
+.. code-block:: bash
+
+   pip install neptune-contrib
+
+
+**STEP 4**
+
+Import neptune, and add your initialization code:
+
+.. code-block:: python3 
+
+   import neptune
+   neptune.init(api_token='ADD-API-TOKEN-HERE',project_qualified_name='risingodegua/spam-classifier')
+
+The code above will initialize your project, and connect your local neptune client to your online account. So that any logs made locally can be synced online.
+
+.. Note::
+   Your API token is a secret, and should be hidden from outsiders. In production environments, it is advisable to add the token to your environment variable. Neptune will automatically pick up the variable from the environment as long as it is called NEPTUNE_API_TOKEN, so you can easily initialize Neptune by running:
+
+.. code-block:: python3
+
+   neptune.init(project_qualified_name='risingodegua/spam-classifier')
+
+
+
+**STEP 5**
+
+Create your first experiment under your project
 
-Installation
-^^^^^^^^^^^^
 .. code-block:: bash
 
-    pip install neptune-contrib
+   neptune.create_experiment(name='spam_exploration')
```

Comment:
I'd change the name to something generic -> remember this is an integration page not a blog post about spam classification with keras.